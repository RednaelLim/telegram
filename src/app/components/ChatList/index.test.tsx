import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { ChatList } from './';

it('ChatList renders correctly', () => {
  const noop = () => false;
  const tree = renderer.create(
    <ChatList
      loading={false}
      hasMore={false}
      loadMore={noop}>
      Hello, world!
    </ChatList>,
  ).toJSON();
  const tree2 = renderer.create(
    <ChatList
      loading={true}
      hasMore={false}
      loadMore={noop}>
      Hello, world!
    </ChatList>,
  ).toJSON();

  expect(tree).toMatchSnapshot();
  expect(tree2).toMatchSnapshot();
});
