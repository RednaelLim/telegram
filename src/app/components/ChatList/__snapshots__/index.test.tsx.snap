// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatList renders correctly 1`] = `
<div
  className="chatlist"
>
  <div
    className="chatlistsearch"
  >
    <input
      className="search"
      placeholder="Search"
      type="text"
    />
    <div
      className="searchButton"
    />
  </div>
  <div
    className="chatbody"
  >
    <div
      className="box"
    >
      Hello, world!
    </div>
  </div>
</div>
`;

exports[`ChatList renders correctly 2`] = `
<div
  className="chatlist"
>
  <div
    className="chatlistsearch"
  >
    <input
      className="search"
      placeholder="Search"
      type="text"
    />
    <div
      className="searchButton"
    />
  </div>
  <div
    className="chatbody"
  >
    <div
      className="box"
    >
      Hello, world!
      <div
        className="chatlist"
      >
        <div
          className="loading"
        >
          Loading...
        </div>
      </div>
    </div>
  </div>
</div>
`;
